SU-SWS/stanford_field_formatters
# abstracted font awesome icon from bean types

# READY FOR REVIEW

# Summary
- Abstracted font awesome field type from bean module.

# Needed By (Date)
- 4/12

# Steps to Test

1. checkout branch, clear cache
2. add font awesome icon field type to any content/entity
3. validate the icon displays as expected.

# See Also
- [Bean Types](https://github.com/SU-SWS/stanford_bean_types/pull/49)
